sources/packages/api/src/types/config.ts

```diff
@@ -8,3 +9,41 @@ export interface IAuthConfig {
   clientId: string;
   openIdConfigurationUrl: string;
 }
+
+/**
+ * BCeID configuration required to execute the SOAP requests
+ * to the BCeID Web Service.
+ */
+export interface BCeIDConfig {
```

Comment:
Rename to IBCeIDConfig

Reply:
We do not have a pattern in place to dictate this and so far it seems that the majority of the interface is not using the "I".